README.md

```diff
@@ -57,6 +57,10 @@ Kubernetes documentation is organized into several categories.
     - in [docs](docs/overview.md)
     - for people who want to run programs on kubernetes
     - describes current features of the system (with brief mentions of planned features)
+  - **Admin Documentation**
```

Comment:
Thanks for noticing this. To make sure this gets in before the deadline, I created a separate PR with this change (#11560)